documentation/modules/creating-migration-quick-plan-vm.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-migration-quick-plan-vm_{context}"]
+= Creating and running a migration plan starting on Providers for virtualization
+
+.Procedure
+
+. In the {ocp} web console, click *Providers for virtualization*.
+. In the row of the appropriate source provider, click the *VMs* entry.
```

Comment:
```suggestion
. In the row of the appropriate source provider, click the *Virtual Machines* entry.
```
and shouldn't it be 'tab' instead of 'entry'? I'm not sure about the terms in the OpenShift console (not that important though)

Reply:
Probably should be "... click *VMs*." We're on the Providers page, the user clicks VMs.


![image](https://github.com/kubev2v/forklift-documentation/assets/60698649/8d668938-f65e-49a5-a090-348be68bd9bb)

Reply:
ah ok, I didn't read it carefully enough, I thought you meant "Virtual Machines" here:
![image](https://github.com/kubev2v/forklift-documentation/assets/6020068/f58a5d43-f4a8-47ff-875e-de99690bbcfc)

Reply:
Well I skipped a step, so I'm glad you asked.